apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,188 @@
+            "governments__title",
+        )
+        return len(set(list(itertools.chain(*qs))))
+
+    @staticmethod
+    def resolve_total_collection_technique(root: AssessmentDashboardStat, info) -> int:
+        return (
+            root.methodology_attribute_qs.values("data_collection_technique").annotate(
+                count=Count("data_collection_technique")
+            )
+        ).count()
+
+    @staticmethod
+    def resolve_assessment_count(root: AssessmentDashboardStat, info):
+        assessment = (
+            root.assessment_registry_qs.values("coordinated_joint")
+            .annotate(count=Count("coordinated_joint"))
+            .order_by("coordinated_joint")
+        )
+
+        return [
+            AssessmentCountType(
+                coordinated_joint=assessment["coordinated_joint"],
+                count=assessment["count"],
+            )
+            for assessment in assessment
+        ]
+
+    @staticmethod
+    def resolve_stakeholder_count(root: AssessmentDashboardStat, info):
+        stakeholder_counts = defaultdict(int)
+
+        organization_type_fields = [
+            "lead_organizations__organization_type__title",
+            "international_partners__organization_type__title",
+            "donors__organization_type__title",
+            "national_partners__organization_type__title",
+            "governments__organization_type__title",
+        ]
+
+        for field in organization_type_fields:
+            stakeholders = root.assessment_registry_qs.values(field)
+            for stakeholder in stakeholders:
+                if organization_type_title := stakeholder.get(field):
+                    stakeholder_counts[organization_type_title] += 1
+        return [
+            StakeholderCountType(
+                stakeholder=org_type_title,
+                count=count,
+            )
+            for org_type_title, count in stakeholder_counts.items()
+        ]
+
+    @staticmethod
+    def resolve_collection_technique_count(root: AssessmentDashboardStat, info):
+        data_collection_technique = (
+            root.methodology_attribute_qs.values("data_collection_technique")
+            .annotate(count=Count("data_collection_technique"))
+            .order_by("data_collection_technique")
+        )
+        return [
+            CollectionTechniqueCountType(
+                data_collection_technique=technique["data_collection_technique"],
+                count=technique["count"],
+            )
+            for technique in data_collection_technique
+        ]
+
+    @staticmethod
+    def resolve_total_multisector_assessment(
+        root: AssessmentDashboardStat, info
+    ) -> int:
+        return sum(1 for a in root.assessment_registry_qs if len(a.sectors) > 1)
+
+    @staticmethod
+    def resolve_total_singlesector_assessment(
+        root: AssessmentDashboardStat, info
+    ) -> int:
+        return sum(1 for a in root.assessment_registry_qs if len(a.sectors) < 1)
```

Comment:
Let's use postgres. Maybe something like `ArrayLenTransform`

Some links to check out:
- https://code.djangoproject.com/ticket/29429
- https://docs.djangoproject.com/en/3.2/_modules/django/contrib/postgres/fields/array/